srcpkgs/syslog-ng/template

```diff
@@ -0,0 +1,88 @@
+# Template file for 'syslog-ng'
+pkgname=syslog-ng
+version=4.9.0
+revision=1
+_jwt_cpp_version=0.7.1
+_opentelemetry_proto_version=1.7.0
+build_style=gnu-configure
+configure_args="--sysconfdir=/etc/syslog-ng --enable-ipv6 --enable-json --enable-manpages --enable-native
+ --enable-python --enable-python-modules --enable-rdrand --disable-geoip2 --disable-java --disable-java-modules
+ --disable-mongodb --disable-python-modules --disable-riemann --disable-systemd
+ $(vopt_if amqp       --enable-amqp)
+ $(vopt_if geoip      --enable-geoip)
+ $(vopt_if http       --enable-http)
+ $(vopt_if linux_caps --enable-linux-caps --disable-linux-caps)
+ $(vopt_if redis      --enable-redis)
+ $(vopt_if smtp       --enable-smtp       --disable-smtp)
+ $(vopt_if sql        --enable-sql)
+ $(vopt_if stomp      --enable-stomp)
+ --with-ivykis=system --with-jsonc=system --with-librabbitmq-client=system --with-python-packages=system"
+hostmakedepends="autoconf autoconf-archive automake bison docbook-xsl flex gperf pkgconf
+ python3 python3-setuptools libtool libxslt tar"
+makedepends=" ivykis-devel json-c-devel glib-devel libcurl-devel libdbi-devel
+ libnet-devel net-snmp-devel pcre2-devel openssl-devel python3-devel
+ $(vopt_if redis hiredis-devel) $(vopt_if amqp rabbitmq-c-devel) $(vopt_if smtp libesmtp-devel)
+ $(vopt_if sql libdbi-drivers-devel) $(vopt_if geoip geoip-devel) $(vopt_if linux_caps libcap-devel)"
+short_desc="Enhanced log daemon, supporting a wide range of input and output methods"
+conf_files="/etc/syslog-ng/syslog-ng.conf"
+maintainer="Komeil Parseh <komeilparseh@disroot.org>"
+license="GPL-2.0-or-later OR LGPL-2.1-or-later"
+homepage="https://www.syslog-ng.com/"
+changelog="https://raw.githubusercontent.com/syslog-ng/syslog-ng/refs/heads/develop/NEWS.md"
+distfiles="https://github.com/syslog-ng/syslog-ng/archive/refs/tags/syslog-ng-${version}.tar.gz
+https://github.com/Thalhammer/jwt-cpp/releases/download/v${_jwt_cpp_version}/jwt-cpp-v${_jwt_cpp_version}.tar.gz
+https://github.com/open-telemetry/opentelemetry-proto/archive/refs/tags/v${_opentelemetry_proto_version}.tar.gz>opentelemetry-proto-v${_opentelemetry_proto_version}.tar.gz"
+checksum="39b2ada5b92830199814a0d5f52a45acc05572016a108f4a8ee3ee20fadba5eb
+ d45894f57437ce45233cfe0e07383c4e1f32f969edfd8df8347e177b13bf74e5
+ 11330d850f5e24d34c4246bc8cb21fcd311e7565d219195713455a576bb11bed"
+
+system_accounts="_syslog"
+
+build_options="amqp geoip http linux_caps redis sql smtp stomp"
+build_options_default="amqp geoip http redis sql stomp"
```

Comment:
Matter of taste, but I'd expect linux_caps to be on by default, but perhaps geoip not (we don't have geoip databases afaiu).